README.md

```diff
@@ -74,11 +74,60 @@ templates/make_manifest openstack-nova my-networking.yml
 bosh -n deploy
 ```
 
+Basic authentication
+-----------------------
+
+You have to create the new password just like for `.htpasswd` file. 
+You can use the Online [Htpasswd Generator](http://www.htaccesstools.com/htpasswd-generator/).
```

Comment:
For security's sake, let's not recommend an online password-handling utility.

Instead, can we write up a small howto on using the standard `htpasswd` utility to create a file and then dump its contents into the manifest?